pkgs/development/python-modules/coreschema/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  fetchPypi,
+  buildPythonPackage,
+  jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "coreschema";
+  version = "0.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01qn9bfkklpjkr0zn6bd5fr372n1jd5p455scw4ap0nl0xh500wm";
+  };
+
+  patchPhase = ''
+    rm -f coreschema/encodings/corejson.py
```

Comment:
Could these at least come with a comment explaining them? Otherwise they're a bit scary to see. Also, if they are only for python 3 it's probably better to only do this conditionally.

Reply:
thanks for the feedback